CHANGELOG.md

```diff
@@ -18,6 +19,7 @@
 
 ### Testing
 * Added gin test for `CellExplorerRecordingInterface` [#488](https://github.com/catalystneuro/neuroconv/pull/488)
+* Added gin test for `CellExplorerSortinggInterface`  [PR #494](https://github.com/catalystneuro/neuroconv/pull/494)
```

Comment:
Can you add a note to these changelog lines that it's about the newer CellExplorer format? (with version info if there's any online since I know there aren't in the files)

Reply:
Sure, I am not sure the changelog is a good place for this though. I will add in both in the docstring and in the changelog.

Reply:
OK, added something. I think there is a confusion here. The format that contains `basename.session.mat` is referred as Cell Explorer. What I am referring as the "old format" here they refer as Buzcode (see one of the last emails that Petersen sent to us). 

So again, this is not a version thing, CellExplorer as they describe it should have a `session.mat` but because we were referring to their buzcode as CellExplorer I am making this distinction between old and new format.

What I added reflects this but feel free to add a diff that you think better reflects the situation.